content/en/docs/languages/java/sdk.md

```diff
@@ -0,0 +1,1382 @@
+---
+title: Manage Telemetry with SDK
+linkTitle: Manage Telemetry with SDK
+weight: 11
+aliases: [sdk-concepts]
+cSpell:ignore: autoconfigured FQCNs Interceptable Logback okhttp
+---
+
+<!-- markdownlint-disable blanks-around-fences -->
+<?code-excerpt path-base="examples/java/configuration"?>
+
+The SDK is the built-in reference implementation of the
+[API](../instrumentation/), processing and exporting telemetry produced by
+instrumentation API calls. This page is a conceptual overview of the SDK,
+including descriptions, links to relevant Javadocs, artifact coordinates, sample
+programmatic configurations and more. See
+**[Configure the SDK](../configuration/)** for details on SDK configuration,
+including
+[zero-code SDK autoconfigure](../configuration/#zero-code-sdk-autoconfigure).
+
+The SDK consists of the following top level components:
+
+- [SdkTracerProvider](#sdktracerprovider): The SDK implementation of
+  `TracerProvider`, including tools for sampling, processing, and exporting
+  spans.
+- [SdkMeterProvider](#sdkmeterprovider): The SDK implementation of
+  `MeterProvider`, including tools for configuring metric streams and reading /
+  exporting metrics.
+- [SdkLoggerProvider](#sdkloggerprovider): The SDK implementation of
+  `LoggerProvider`, including tools for processing and exporting logs.
+- [TextMapPropagator](#textmappropagator): Propagates context across process
+  boundaries.
+
+These are combined into [OpenTelemetrySdk](#opentelemetrysdk), a carrier object
+which makes it convenient to pass fully-configured
+[SDK components](#sdk-components) to instrumentation.
+
+The SDK comes packaged with a variety of built-in components which are
+sufficient for many use cases, and supports
+[plugin interfaces](#sdk-plugin-extension-interfaces) for extensibility.
+
+## SDK plugin extension interfaces
+
+When built-in components are insufficient, the SDK can be extended by
+implementing various plugin extension interfaces:
+
+- [Sampler](#sampler): Configures which spans are recorded and sampled.
+- [SpanProcessor](#spanprocessor): Processes spans when they start and end.
+- [SpanExporter](#spanexporter): Exports spans out of process.
+- [MetricReader](#metricreader): Reads aggregated metrics.
+- [MetricExporter](#metricexporter): Exports metrics out of process.
```

Comment:
nit: is it always out of process, or typically out of process?
(Eg: InMemoryExporter)

Reply:
Typically out of process. Although InMemoryExporter is a special case, and maybe the only exception to the rule. It may be better to generalize to avoid burdening readers with understanding the exception cases, given that this is a terse summary sentence.

Reply:
(There was some zpages thing in the past as well.)

Reply:
can this conversation be resolved?